src/db/seeders/20220514012040-sample-schools.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+module.exports = {
+  up: async queryInterface => {
+    return await queryInterface.bulkInsert('schools', [
+      {
+        name: 'Dulwich College Shanghai Pudong',
```

Comment:
Change these names to just their regions: 
Shanghai Pudong, 
Singapore

Reply:
Also please add a Beijing hahaha Shanghai was our rival school

Reply:
HAHA same, they beat us in vball .-.